### DISCUSSION TITLE
Flashcards clarifications

### DISCUSSION THREAD

**Discussion by Ifeoluwapo49** [st3v3nmw/obsidian-spaced-repetition#558] [Q&A]
How do I create multiple cards at once without having to repetitively use the tag at the beginning of each card? 

Also, for cloze cards, how do I specify which clozes I want to be obscured for each card? The Anki {{c1::}} format doesn't work.

**Comment by st3v3nmw** [comment]
> How do I create multiple cards at once without having to repetitively use the tag at the beginning of each card?

Tagging the note is enough to match all flashcards inside that note.

> Also, for cloze cards, how do I specify which clozes I want to be obscured for each card? The Anki {{c1::}} format doesn't work.

That isn't currently supported. We're tracking the feature in https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/93
